docs/eventing/event-registry.md

```diff
@@ -69,214 +64,199 @@ spec:
-  are two topics). You can see that in the registry example output from the
-  previous sections.  
-
-
+    As Manual Registration might be tedious and error-prone, we also support automatic registration of EventTypes. 
+    Herein, the creation of the EventTypes is done upon instantiation of an Event Source. 
+    We currently support automatic registration of EventTypes for the following Event Sources:
+
+    - CronJobSource
+    - ApiServerSource
+    - GithubSource
+    - GcpPubSubSource
+    - KafkaSource
+    - AwsSqsSource
+    
+    Let's look at an example, in particular, the KafkaSource sample we used to populate the registry in our testing 
+    cluster. Below is what the yaml looks like. 
+
+    ```yaml
+    apiVersion: sources.eventing.knative.dev/v1alpha1
+    kind: KafkaSource
+    metadata:
+      name: kafka-sample
+      namespace: default
+    spec:
+      consumerGroup: knative-group
+      bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
+      topics: knative-demo,news
+      sink:
+        apiVersion: eventing.knative.dev/v1alpha1
+        kind: Broker
+        name: default       
+    ```
+    
+    If you are interested in more information regarding configuration options of a KafkaSource, please refer to the 
+    [KafKaSource example](https://github.com/knative/eventing-sources/tree/master/contrib/kafka/samples).
+    
+    For this discussion, the relevant information from the yaml above are the `sink` and the `topics`. 
+    We observe that the `sink` is of kind `Broker`. We currently only support automatic creation of EventTypes for Sources 
+    instances that point to Brokers.
+    Regarding `topics`, this is what we use to generate the EventTypes `source` field, which is equal to the CloudEvent 
+    source attribute. 
+
+    When you `kubectl apply` this yaml, the KafkaSource `kafka-source-sample` will be instantiated,
+    and two EventTypes will be added to the registry (as there are two topics). You can see that in the registry 
+    example output from the previous sections.  
+        
 ## What's next
 
-We suggest the reader to experiment in her own cluster with the different Event
-Sources listed above. The following links might help to get you started.
+We suggest the reader to experiment in her own cluster with different Event Sources. The following 
+links might help to get you started. 
 
-1. [Installing Knative](./install/README.md) in case you haven't already done
-   so.
-1. [Getting started with eventing](./eventing) in case you haven't read it.
-1. [Knative code samples](./samples/) is a useful resource to better understand
-   some of the Event Sources (remember to point them to a Broker if you want
-   automatic registration of EventTypes in the registry).
+1. [Installing Knative](./install/README.md) in case you haven't already done so.
+1. [Getting started with eventing](./eventing) in case you haven't read it. 
```

Comment:
"Getting started with eventing" is a broken link.

Reply:
Changed it to ./README.md, thanks!